onnx/cpp2py_export.cc

```diff
@@ -372,6 +372,12 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
     checker::check_node(proto, ctx, lex_ctx);
   });
 
+  checker.def("check_function", [](const py::bytes& bytes, const checker::CheckerContext& ctx) -> void {
+    FunctionProto proto{};
+    ParseProtoFromPyBytes(&proto, bytes);
+    checker::check_function(proto, ctx, checker::LexicalScopeContext());
```

Comment:
I wonder why the last parameter (lexical scope context) is not exposed to the Python interface? I realize that the same problem exists in the other existing methods, but maybe we should consider exposing it (while maintaining backward compatibility).